src/ray/raylet/worker_pool_test.cc

```diff
@@ -96,9 +96,10 @@ class WorkerPoolMock : public WorkerPool {
  public:
   explicit WorkerPoolMock(instrumented_io_context &io_service,
                           const WorkerCommandMap &worker_commands)
-      : WorkerPool(io_service, NodeID::FromRandom(), "", POOL_SIZE_SOFT_LIMIT, 0,
-                   MAXIMUM_STARTUP_CONCURRENCY, 0, 0, {}, nullptr, worker_commands,
-                   []() {}, 0, [this]() { return current_time_ms_; }),
+      : WorkerPool(
```

Comment:
Are these C++ changes intended? It seems unrelated from the PR description

Reply:
Maybe you want to `git pull . upstream/master`.

Reply:
done, thanks